CITATION

```diff
@@ -1,34 +1,30 @@
 If you use Gammapy for work/research presented in a publication (whether
 directly, or as a dependency to another package), we recommend and encourage
-the following acknowledgment:
-
-  This research made use of Gammapy, a community-developed core Python package for gamma-ray astronomy
-  (Deil et al., 2017).
-
-where (Deil et al., 2017) is a citation to this paper:
-
-  https://ui.adsabs.harvard.edu/abs/2017ICRC...35..766D
-
-We encourage you to also include citations to the paper in the main text
-wherever appropriate.
+the following citation scheme:
+- refer to the Gammapy paper as general citation: A&A, 678, A157 (2023) - the bibtex is given bellow
```

Comment:
maybe its better to explicitly mention that oner should cite paper AND zenodo version, not either or